showcase-docs/getting-started.md

```diff
@@ -2,6 +2,81 @@
 
 There are several different methods for running the Backstage Showcase app today. We currently have support for running the application locally, using a helm chart to deploy to a cluster, and manifests for deployment using ArgoCD.
 
+## Telemetry collection
+
+This software may enable telemetry data collection through the [`@janus-idp/backstage-plugin-analytics-provider-segment`](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/analytics-provider-segment) plugin in its default configuration to enhance user experience while prioritizing privacy:
+
+- **Anonymized-focused configuration**:
+
+  - IP addresses are anonymized (`maskIP: true`), and recorded as `0.0.0.0`.
+  - `anonymousId` used for tracking is a hash derived from the user's username.
+
+- **Data Collection Overview**:
+  - **Events Tracked**: Page visits, clicks on links or buttons.
+  - **Common Data Points for All Events**:
+    - User-related info: Locale, timezone, userAgent (browser and OS details).
+    - Page-related info: Title, Category, Extension name, URL, path, referrer, search parameters.
+
+This ensures a thorough understanding of user interactions with the application while maintaining user anonymity and privacy.
+
+The data will be used only for internal analysis and product improvement.
+
+If you wish to enable telemetry data collection, follow the steps below.
+
+### Enable Telemetry Using Helm Chart
+
+To turn on the telemetry while using the Helm chart, you need to enable Segment provider plugin in your Helm `values.yaml` file by adding the following configuration:
+
+```yaml
+global:
+  dynamic:
+    plugins:
+      - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
+        disabled: false
+```
```

Comment:
Just for clarity: does the user need to add the entire YAML code or just need to edit the value of the `disabled` parameter?

Reply:
user needs to add the
```
      - package: './dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment'
        disabled: false
``` 
into `plugins` section, which will mostly likely already exist as it is part of the default configuration